src/registrar/templates/home.html

```diff
@@ -13,7 +13,7 @@
   {% block messages %}
     {% include "includes/form_messages.html" %}
   {% endblock %}
-  <h1>Manage your domains</h2>
+  <h1>Manage your domains - TESTING TESTING 123</h2>
```

Comment:
**I will remove this before I merge.**

If you see this in the header when you first log in, that means you're on the most updated site. Running into some sandbox and deployment issues right now, so please just ignore this.